assets/js/apps/search/components/PreviewCard/EntryPreviewCard.stories.jsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions';
+import EntryPreviewCard from './EntryPreviewCard';
+
+export default {
+  component: EntryPreviewCard,
+  title: 'EntryPreviewCard',
+  decorators: [story => <div style={{ 
+    border: '2px solid black'
+  }}>{story()}</div>],
+};
+
+const project = {
+  "type": "project",
+  "url": "idk/",
+  "counts": {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
    "counts": {
```